app/screens/post_options/components/options/pin_channel_option.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo} from 'react';
+
+import {t} from '@i18n';
+
+import BaseOption from './base_option';
+
+type PinChannelProps = {
+    isPostPinned: boolean;
+}
+
+//fixme: wire up handlePinChannel
+const PinChannelOption = ({isPostPinned}: PinChannelProps) => {
+    const handlePinPost = () => null;
+    const handleUnpinPost = () => null;
+
+    const config = useMemo(() => {
```

Comment:
No need for memo. The information passed down to the next component are base types. The only one that could use usememo/useCallback is onPress.

Reply:
fixed